index.php

```diff
@@ -57,6 +61,9 @@ function goShabad ($n, $h) {
 	case 's':
 		goShabad($path[1], $path[2]);
 		break;
+	case 'sg':
+		go('$sttm/sundar-gutka');
```

Comment:
I don't think PHP would interpolate $sttm variable here if you use a single quote.